app/scripts/modules/core/src/utils/index.ts

```diff
@@ -18,3 +18,4 @@ export * from './timeFormatters';
 export * from './unicodeBase64';
 export * from './uuid.service';
 export * from './workerPool';
+export * from './json/filterObjectValues';
```

Comment:
just saw this is in `core` , let me know if you want me to add this in a separate PR